CODE_OF_CONDUCT.md

```diff
@@ -17,7 +15,7 @@ A supplemental goal of this Code of Conduct is to increase open [source/culture/
 
 Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
 
```

Comment:
To provide equal access of information to all, the Mobile Development Club shall refrain from actions and methods of broadcasts that unnecessarily exclude certain demographics or individuals from accessing information released and announced by the RSO (examples against this clause include but are not limited t0 making announcements solely to major and department-specific communication mailing-lists and other communication groups OR to explicitly state or conduct outreach in a fashion that suggests that the RSO is "exclusive to people in computing").